src/main/java/frc/robot/Robot.java

```diff
@@ -66,6 +123,9 @@ public void autonomousPeriodic() {
 
     @Override
     public void teleopInit() {
+        // climber.zeroClimber();
+        drive.setHeadingCorrection(true);
+        manager.intake.pivotMotor.setIdleMode(IdleMode.kBrake);
```

Comment:
the pivotMotor on the intake should be `private`, and this should be controlled with a public method on Intake